scripts/actions/remove-completed-batch.js

```diff
@@ -17,4 +24,63 @@ const saveRemainingBatches = async () => {
   );
 };
 
+/**
+ * Gets the fileUris from the deserializedBatchUids, takes the fileUris and fetches
+ * the contextUids, and then the contextUids are deleted for deserialized batches
+ * @param {Array} deserializedBatchUids
+ * @returns {Promise}
+ */
+const removePageContext = async (deserializedBatchUids) => {
+  const accessToken = await getAccessToken();
+
+  const fileUris = await deserializedBatchUids.map(async (batchUid) => {
```

Comment:
I _think_ we might want to `await Promise.all(deserializedBatchUids.map(async ...` here since the `map` will return an array of promises

Reply:
yes very true! thank you